nixos/modules/virtualisation/libvirtd.nix

```diff
@@ -129,6 +129,60 @@ let
       };
     };
   };
+
+  hooksModule = types.submodule {
+    options = {
+      daemon = mkOption {
+        type = types.attrsOf types.str;
+        default = {};
+        description = lib.mdDoc ''
+          Hook that will be placed under $SYSCONFDIR/libvirt/hooks/daemon
```

Comment:
Could we use the actual value instead of `$SYSCONFDIR`?

Reply:
Done.